spiceaidocs/docs/reference/spicepod/datasets.md

```diff
@@ -159,6 +159,12 @@ Must be of the form `SELECT * FROM {name} WHERE {refresh_filter}`. `{name}` is t
 - Queries for data that have been filtered out will not fall back to querying against the federated table.
 :::
 
+## `acceleration.refresh_period`
+
+Optional. Applies the provided recent period filter when fetching data from the source. Combine with `time_column` and `time_format` to determine the recent period to be applied. Only supported for `full` refresh_mode datasets.
```

Comment:
`provided recent period filter` - this isn't very clear. What is the recent period filter? Is this retention period?